main.js

```diff
@@ -0,0 +1,143 @@
+/* eslint-disable linebreak-style */
+// /* eslint-disable no-undef
+// eslint-disable no-use-before-define
+/* eslint-disable linebreak-style */
+// create the book constructor
+
+function Book(title, author, isbn) {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.